.github/workflows/version_gallery.yml

```diff
@@ -0,0 +1,32 @@
+name: Past PyNWB Version
```

Comment:
is there a reason you are setting this up as a cron job instead of just adding it to the PR triggered workflow?

Reply:
Multiplicity - If I include it in the main workflow we'd have (number of platforms) x (number of Python versions) x (number of PyNWB versions), which gets a bit out of hand.

This instead constrains to `ubuntu-latest` + Python 3.9 and then goes back as far in the version history as desired.